package.json

```diff
@@ -36,7 +36,7 @@
     "lodash.get": "^4.4.2",
     "react": "^16.6.0",
     "react-dom": "^16.6.0",
-    "rx-lite": "^4.0.8"
+    "rxjs": "^6.3.3"
```

Comment:
Remove this, pelase, since it added in other MR

Reply:
Removed